### ISSUE TITLE
NVDA interrupts speech when a new Teams message arrives

### ISSUE THREAD

**Issue by munawarb** [nvaccess/nvda#13081]
### Steps to reproduce:

1. Run Microsoft Teams.
2. Switch to another window by pressing ALT+TAB.
3. Begin reading a long piece of text.
4. Expect a new Teams message.
5. Upon arrival of the new message, NVDA will stop reading whatever it was reading in the new window to notify the user that a new message has arrived.
6. If multiple messages arrive, it is very difficult to read content because speech keeps getting interrupted.

### Actual behavior:

Whether or not speech is interrupted (or even if Teams announcements are read at all) should be configurable from a Teams options dialog provided by NVDA.

### Expected behavior:

NVDA's speech interrupt should be configurable. If it was, in the situation listed above, I should be able to instruct it to ignore new Teams notifications unless it's an incoming call (this is the level of customization I'd expect from a well supported product.)

**Comment by seanbudd** [comment]
Welcome @,

We are unable to process this issue as it stands.

This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by XLTechie** [comment]
@seanbudd
Your attempt to welcome/reference @munawarb does not seem to have worked; there 
was no username in your @ reference.
